tensorflow/python/ops/histogram_ops.py

```diff
@@ -136,6 +141,9 @@ def histogram_fixed_width(values,
   >>> hist.numpy()
   array([2, 1, 1, 0, 2], dtype=int32)
   """
+  if nbins < 0:
```

Comment:
This one is handled in the C++ implementation, we don't need it here.

Reply:
Removed this condition.